File: Models/StudentAnswer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EducationalSoftwareServer.Models
{
    public class StudentAnswer
    {
        public int StudentAnswerId { get; set; }
        public int TestId { get; set; }
        public int StudentId { get; set; }
        public int QuestionId { get; set; }
        public string StudentResult { get; set; }
    }
}

File: Client/EducationalSoftwareClient/src/app/models/student-answer.ts
export class StudentAnswer{
    studentAnswerId?: number;
    testId?: number;
    studentId?: number;
    questionId?: number;
    studentResult? : string;

    constructor(){
        this.studentAnswerId = 0;
        this.testId = 0;
        this.studentId = 0;
        this.questionId = 0;
        this.studentResult = "";
    }
}